requirements_dev.txt

```diff
@@ -6,7 +6,7 @@ Sphinx>=2.4.3
 twine>=3.1.1
 
 contextvars==2.4;python_version<"3.7"
-fastapi>=0.63.0
+fastapi>=0.65.1
```

Comment:
First, I would not change version of FastAPI in requirements. The only big breaking change we know so far is removal of UJSONResponse in newer versions of Starlette. Luckily, that is easy to fix to support both 0.63 and latest 0.65.1 version. And we've had issues in the past when we pinned down version by specifying upper bound, so I think we shouldn't do it now as well (I mean that we could NOT just >=0.63.0, <0.66).

Reply:
Also, requirements for project itself (NOT for development) are specified in setup.py

Reply:
sorry, didn't get it, what is the problem with having fixed dependency on 0.65.1? Should I fix the concrete version with `fastapi==0.65.1`?

Reply:
Not really a problem, but I think we can continue support for fastapi from 0.63.0, since we are now importing plain `JSONResponse`, that is present in 0.63.0 as well as in newer versions (given that we still implement our own `UJSONResponse`).

Reply:
oh, okay, I thought in context of broken security check and assumed that version should be bumped to 0.65.1, will fix